test/e2e/specs/fakerp/metricsbridge.go

```diff
@@ -0,0 +1,133 @@
+package fakerp
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net"
+	"net/http"
+	"regexp"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+	"github.com/prometheus/client_golang/api"
+	"github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/openshift-azure/test/sanity"
+)
+
+type authorizingRoundTripper struct {
+	http.RoundTripper
+	token string
+}
+
+func (rt authorizingRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Set("Authorization", "Bearer "+rt.token)
+	return rt.RoundTripper.RoundTrip(req)
+}
+
+var _ = Describe("Metricsbridge E2E check configured queries [Fake][EveryPR]", func() {
+	//queries which usually return no datapoints, but have been verified as valid manually
+	manuallyVerifiedQueries := map[string]bool{
+		"kubelet_runtime_operations_errors != 0": true,
+		"kubelet_docker_operations_errors != 0":  true,
+	}
+
+	BeforeEach(func() {
+	})
+
+	It("should be possible to get data for each metric in defined queries", func() {
+		By("getting the configMap")
+		mbConfigMap, err := sanity.Checker.Client.Admin.CoreV1.ConfigMaps("openshift-azure-monitoring").Get("metrics-bridge", meta_v1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		var mbConfig struct {
```

Comment:
consider centralizing the metrics bridge config somewhere and share that between here and the metrics bridge client

Reply:
reduced the config struct to what I need after consulting with Charles.